packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.tsx

```diff
@@ -215,6 +215,21 @@ function getPaymentMethodTitle(
             },
         };
 
+
+        if (method.gateway === PaymentMethodId.BlueSnapDirect) {
```

Comment:
Can we update test for these conditions?

Reply:
done